content/en/docs/concepts/wasm/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: WebAssembly (Wasm) Plugins
```

Comment:
WASM isn't a concept - its a technology.

The 'concept' here is 'Extensibility' can we retitle ?